src/GraphBLAS-sharp.Backend/Objects/Matrix.fs

```diff
@@ -7,30 +7,31 @@ type MatrixFormat =
     | COO
     | CSC
 
-type Matrix<'a when 'a: struct> =
-    | MatrixCSR of CSRMatrix<'a>
-    | MatrixCOO of COOMatrix<'a>
-    | MatrixCSC of CSCMatrix<'a>
+[<RequireQualifiedAccess>]
+type ClMatrix<'a when 'a: struct> =
+    | CSR of ClCSRMatrix<'a>
+    | COO of ClCOOMatrix<'a>
+    | CSC of ClCSCMatrix<'a>
 
     member this.RowCount =
         match this with
-        | MatrixCSR matrix -> matrix.RowCount
-        | MatrixCOO matrix -> matrix.RowCount
-        | MatrixCSC matrix -> matrix.RowCount
+        | ClMatrix.CSR matrix -> matrix.RowCount
```

Comment:
Can I propose to refactor this file like [this](https://gist.github.com/gsvgit/41f39cc3d6bf2562aeda15363ee48f0d) to avoid ugly `ClCSRMatrix`?

Reply:
May we do the same for [vector](src/GraphBLAS-sharp.Backend/Objects/Vector.fs)?